hack/validate-deps.sh

```diff
@@ -21,7 +21,7 @@ cd "${T}/"
 glide up -v
 
 
-if [[ -n "$(git status --porcelain)" ]]; then
+if [[ -n "$(git status --porcelain | sed  "/M glide.lock/d")" ]]; then
```

Comment:
still needed?

Reply:
if we gonna use it in any CI - yes. If we will run locally only - no